tests/PluginTest.php

```diff
@@ -152,6 +153,7 @@ public function testForwardEventWithRecognizedCommand()
             'foo',
             'bar',
             $eventName,
+            array(),
```

Comment:
In addition to these, a test case should be added that actually verifies that custom parameters are processed correctly.